src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -188,9 +210,18 @@ protected ConstantValue MakeDefaultExpression(DiagnosticBag diagnostics, Binder
             BoundExpression valueBeforeConversion;
             var convertedExpression = binderForDefault.BindParameterDefaultValue(defaultSyntax, parameterType, diagnostics, out valueBeforeConversion);
 
+            if (!binder.IsSemanticModelBinder && (this.ContainingSymbol as MethodSymbol)?.MethodKind != MethodKind.LocalFunction)
+            {
+                // It is okay to call this multiple times (in a multithreaded environment),
+                // as the only thing VariableUsePass is add/remove members from sets,
+                // which can be performed multiple times with no further effect.
+                new VariableUsePass(this.DeclaringCompilation.SourceAssembly).Visit(convertedExpression);
```

Comment:
>new VariableUsePass(this.DeclaringCompilation.SourceAssembly).Visit(convertedExpression); [](start = 16, length = 89)

I am not sure why this code was moved. Even though it is OK to do this multiple times, we shouldn't do that if it can be easily avoided.